phage_variants_search.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+#alias "java -jar /home/nvaulin/Documents/Trimmomatic-0.39/trimmomatic-0.39.jar" trimmomatic
+
+# define samples and other necessary parameters
+declare -a SAMPLES=("AI-69_S60" "AI-70_S61" "AI-71_S62" "AI-72_S63" "AI-73_S64")
+REF_GENOME="T5"
+THREADS="8"
+
+# define directories
+QUALITY_DIR=Data/quality_reports
+TRIMMING_DIR=Data/trimmed_reads
+MAPPED_DIR=Data/mapped
+MAPPED_SORTED_DIR=Data/mapped_sorted
+MAPPED_STAT_DIR=Data/mapped_statistics
+CALLING_DIR=Data/calling_files
+
+# create directories
+mkdir ${QUALITY_DIR} ${TRIMMING_DIR}
+mkdir ${MAPPED_DIR} ${MAPPED_SORTED_DIR} ${MAPPED_STAT_DIR}
+mkdir ${CALLING_DIR}
+
+# index reference
+REFERENCE=Data/reference/${REF_GENOME}_sequence.fasta
+bwa index ${REFERENCE}
+
+#iterate over samples
+for SAMPLE in ${SAMPLES[@]}; do
```

Comment:
Yeah, that is bash scripting, thank you!

Also, one could use group renaming to delete the 'S60' etc part so we will have just 'AI-NN' and we can simply iterate not through the array of samples (which can be huge) but over the number from 69 to 73 - more scaleble for big samplings

Or, another option not to write the real sample name - to iterate over the files in the folder:
```bash 
for file in data/raw_reads/*.fastq
```

in this case to extract the sample name (for the output files naming) one could use prefix and postfix removing in bash:
```bash
# delete the postfix
temp=${file%%_R1_001.fastq}
# delete the prefix
sample=${temp##data/raw_reads/}
```